app-native/src/audio.rs

```diff
@@ -56,11 +61,6 @@ pub fn audio_start<'a>(
   let latency = output_info.default_low_output_latency;
   let output_params = StreamParameters::<f32>::new(def_output, CHANNELS, INTERLEAVED, latency);
 
-  let audio_config = studio_lock
-    .read()
-    .map_err(|_err| AudioError::StudioConfig)
-    .map(|studio| studio.config().audio.clone())?;
-
   // Check that the stream format is supported.
   let sample_rate = audio_config.sample_rate as f64;
```

Comment:
Why this casting? Why not to set the types properly in `audio_config`?

Reply:
Using an f64 for a sample rate is something special to port audio. Sample rates are usually integers, that's why the config only allows integers.